### ISSUE TITLE
Whatsapp allows using their brand in their name, as long as you follow their guidelines

### ISSUE THREAD

**Issue by GlowMLG** [WhatsApp/stickers#190]
I emailed Whatsapp support about the last issues with naming apps with "Whatsapp" in the name. [Here you have the mail](https://i.imgur.com/Doixeli.png). They seem to be sure it's a hoax covered in [this article](https://faq.whatsapp.com/en/general/28030005). 

They may have been confused and thought I was talking about a hoax in chat, not when making a stickers app so I've sent them another to confirm.

Edit: Turns out that you must use their name and branding correctly in order to put it in the title. Visit [Whatsapp Branding Resources](https://whatsappbrand.com/) for more information

**Comment by 0566055455** [comment]
So what should I say to google devs to remove the suspension?

**Comment by GlowMLG** [comment]
Do you have any more info on who suspended the app? Was it Whatsapp or a different brand?

**Comment by 0566055455** [comment]
This is the email of who suspended the app
18097@enfappdetex.com

is it official from WhatsApp?

**Comment by GlowMLG** [comment]
Doesn't seem like so to be honest

**Comment by 0566055455** [comment]
I think I will contact appdetex and confirm if this email is from them

**Comment by GlowMLG** [comment]
Okay keep us in touch

**Comment by 0566055455** [comment]
Just received an email from appdetex, they sent me this link https://whatsappbrand.com/

**Comment by GlowMLG** [comment]
Makes sense, thanks for telling us

**Comment by MarsVard** [comment]
Can this be confirmed by any of the repo maintainers?

**Comment by kaditya679** [comment]
Let me tell you the solution & reason for suspension.
first let me tell you guys that you can use whatsapp in you app but there is a condition in that which is you must not put it in first word of your app name.
for eg:- WhatsApp Stickers (violets)
but you can go with like "Sticker for whatsapp", Sticker packs - WhatsApp.... so on.

hope you got it & this will help.

**Comment by 0566055455** [comment]
Just work up to be surprised by an account termination.
2 suspensions from Whatsapp for using their name in the title.

I was using xxx for Whatsapp
and *** stickers - Whatsapp

If anyone here could help me please contact me

**Comment by 0566055455** [comment]
Not sure if I used uppercase A, But if I didnt yeah I deserve to be terminated. sarcasm

**Comment by 0566055455** [comment]
Just confirmed that I was using "WhatsApp" in the title.

https://imgur.com/a/elVLFGU

Maybe we are not allowed to use a dash?
Can any

**Comment by MarvinJWendt** [comment]
Please take a look at the LICENSE file in this repo.

> Neither the name WhatsApp nor the names of its contributors may be used to
   endorse or promote products derived from this software without specific
   prior written permission.

Please also take a look at #150 and #253 

**TL;DR: It is currently not allowed to use WhatsApp in an apps name, doing so will result in a suspension.**
(I made the same mistake, maybe in the future things will work out better)

**Comment by diepox** [comment]
> Just confirmed that I was using "WhatsApp" in the title.
> 
> https://imgur.com/a/elVLFGU
> 
> Maybe we are not allowed to use a dash?

@0566055455 Yes you need to do " XXX Stickers for WhatsApp" not "XXX - WhatsApp"

**Comment by Zandor300** [comment]
@diepox Did you completely ignore @MarvinJWendt's message?

cc @0566055455

**Comment by diepox** [comment]
@Zandor300 Sorry, didn't saw XD

**Comment by simonzhexu** [comment]
Read the read me for Android. Including WhatsApp in your app name is not suggested.

**Comment by MarvinJWendt** [comment]
Hi @immujahidkhan, we cannot give legal advice here. Please consult the WhatsApp legal team for such questions.
